app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -1093,4 +1083,131 @@ private void shareMessage(String message) {
             currContext.startActivity(sendIntent);
         }
     }
+
+    private class SearchResultsAdapter extends RecyclerView.Adapter<SearchResultHolder> {
+        private LayoutInflater inflater;
+        private Context context;
+        private List<Datum> datumList;
+        private Realm realm;
+
+
+        private SearchResultsAdapter(Context context, List<Datum> datumList) {
+            this.context = context;
```

Comment:
Also, no need to pass the context over here, you can do 
 `context = parent.getContext();`
 inside your `onCreateViewHolder()` method.